_data/projects/lucene.net.yml

```diff
@@ -0,0 +1,13 @@
+name: Apache Lucene.Net
+desc: A port of the Lucene fulltext search engine library, written in C# and targeted at .NET runtime users.
+site: http://lucenenet.apache.org/
+tags:
+- .NET
+- C#
+- full-text
+- fulltext
+- lucene
+- search
+upforgrabs:
+- name: get involved
+- link: https://issues.apache.org/jira/browse/LUCENENET/
```

Comment:
Is there a more specific URL you could provide here to help users navigate straight to a list of issues which are good for new contributors?

I was hoping to see a label here that might suit:

![screen shot 2015-04-22 at 5 15 04 pm](https://cloud.githubusercontent.com/assets/359239/7288215/2f470678-e913-11e4-8875-15b997c8b7ff.png)